src/pcrlock/pcrlock.c

```diff
@@ -4911,6 +4916,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "     --force                  Write policy even if it matches existing policy\n"
                "     --entry-token=machine-id|os-id|os-image-id|auto|literal:…\n"
                "                              Boot entry token to use for this installation\n"
+               "     --strict                 All PCRs for which components exist must be predicted\n"
```

Comment:
maybe say "Require that PCRs configured via --pcr= included in policy" or so?

i.e. i mean, that's what this is about, that the selected PCRs are really in the policy